entrypoint.sh

```diff
@@ -8,8 +8,13 @@ export CERT_GROUP_ID
 CURRENTDATE=$(date +'%Y-%m-%d %T')
 echo "${CURRENTDATE}: Docker container has been started. You can find the log file at /var/log/cron.log"
 
+# Run it now
+if [ "$ON_START" == 1 ]; then
+    /app/cert-export.sh
+fi
+
 # Setup a cron schedule
-echo "${CRON_TIME} /app/traefik-ssl-certificate-exporter --source /app/traefik/acme.json --dest /app/certs/ --owner ${CERT_OWNER_ID} --group ${CERT_GROUP_ID} >> /var/log/cron.log 2>&1
+echo "${CRON_TIME} /app/cert-export.sh >> /var/log/cron.log 2>&1
```

Comment:
I've tested this docker image, but you need to change the cronjob else cron cannot determine the ENV vars.
Change it to:
```bash
# Setup a cron schedule
echo "CERT_OWNER_ID="${CERT_OWNER_ID}"
CERT_GROUP_ID="${CERT_GROUP_ID}"
${CRON_TIME} /app/cert-export.sh >> /var/log/cron.log 2>&1
# This extra line makes it a valid cron" > scheduler.txt
```

Reply:
It should be OK now.